.travis.yml

```diff
@@ -20,10 +20,6 @@ env:
 
 matrix:
   fast_finish: true
-  allow_failures:
-    - env: EMBER_TRY_SCENARIO=ember-release
-    - env: EMBER_TRY_SCENARIO=ember-beta
-    - env: EMBER_TRY_SCENARIO=ember-canary
```

Comment:
I think it makes sense to leave ember-canary here, that's the default from the blueprint. Happy to support beta and release though.